src/BloomExe/web/controllers/ApiRequest.cs

```diff
@@ -131,6 +131,13 @@ public static bool Handle(EndpointRegistration endpointRegistration, IRequestInf
 					throw new ApplicationException(string.Format("The EndpointHandler for {0} never called a Succeeded(), Failed(), or ReplyWith() Function.", info.RawUrl.ToString()));
 				}
 			}
+			catch (System.IO.IOException e)
+			{
+				var shortMsg = String.Format(L10NSharp.LocalizationManager.GetDynamicString("Bloom", "WebServer.Warning.CannotAccessFile", "Cannot access {0}"), info.RawUrl);
```

Comment:
I see that people have been using this WebServer.Warning, but it was a mistake. The "paths" are there to help the translator; they should be named after things in the UI, not our internal architecture. Let's change this to "Errors.CannotAccessFile". The fact that we consider it a warning, not an error, is, I expect, irrelevant to users and translators.